vcbuild.bat

```diff
@@ -530,7 +530,7 @@ if not defined NODE_VERSION (
 if not defined DISTTYPE set DISTTYPE=release
 if "%DISTTYPE%"=="release" (
   set FULLVERSION=%NODE_VERSION%
-  exit /b 0
+  goto distexit
```

Comment:
Agreed that the set repetition is also not great. If you guys prefer it this way then let's do that! I just found the `goto :EOF` as the last line confusing, but hopefully long-term we can move to the Powershell promised land anyway.